Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV CYPRESS_INSTALL_BINARY=0
 
 RUN --mount=type=cache,target=/root/.npm/_cacache/ \
     --mount=type=cache,target=/root/.local/share/pnpm/store \
-    npm install -g pnpm@9 && \
+    npm install -g pnpm@9.1.0 && \
```

Comment:
Since maintaining the specific version here would be more of an overhead

can you revert this and also add the following line?
https://github.com/rotki/rotki/blob/7f4a194d0aefcc2b9619f2ae4f6c481aeaacf78e/Dockerfile#L5